app/serializers/search_and_compare/course_serializer.rb

```diff
@@ -63,5 +62,53 @@ def get_subjects
         }
       end
     end
+
+    # Campuses_related_Mapping
+    attribute(:Campuses)                              { get_campuses }
+    # using server time not utc, so it's local time?
+    attribute(:ApplicationsAcceptedFrom)              { object.applications_open_from.to_date.strftime('%Y-%m-%dT%H:%M:%S') }
+    attribute(:HasVacancies)                          { object.has_vacancies? }
+    def get_campus_default_value
```

Comment:
In Ruby we don't typically use `get_` to signal a getter ... this ain't C#!

Reply:
Also, I think these should be private.

Reply:
`campus_default_value` ?

Reply:
Ya, that's the way we usually do it, ya ... call it what it is. :)